iOS/ViewControllers/Courses/CourseListViewController.swift

```diff
@@ -177,6 +230,26 @@ extension CourseListViewController: UISearchControllerDelegate {
             statusBarBackground.autoresizingMask = [.flexibleWidth]
             self.statusBarBackground = statusBarBackground
         }
+
+        self.updateSearchFilterContainerHeight(isSearching: true)
+        self.collectionViewLayout.invalidateLayout()
+
+        // swiftlint:disable:next trailing_closure
+        UIView.animate(withDuration: 0.25, delay: 0, options: .curveEaseInOut, animations: {
```

Comment:
You could add `[unowned self]` here. I think it's good to explicitly add capture rule for `self` in closure by default. It forces you to think about it, and can help prevent memory leaks. 
In my experience, wrong capture in blocks was the no1 cause of memory leaks.

Reply:
Good catch!